.github/workflows/functional-test.yaml

```diff
@@ -559,6 +560,30 @@ jobs:
             --subscription ${{ secrets.INTEGRATION_TEST_SUBSCRIPTION_ID }} \
             --name ${{ env.AZURE_TEST_RESOURCE_GROUP }} \
             --yes --verbose
+      # TODO add once tested: if: failure() && steps.publish-tf-recipes.outcome == 'failure' 
```

Comment:
Would it be bad if we just always included these logs?

Reply:
No it wouldn't. We wanted to track recipe upload failures specifically, but we could just track the events whenever they're uploaded